Cargo.toml

```diff
@@ -294,7 +294,7 @@ siphasher = "0.3.11"
 smpl_jwt = "0.7.1"
 socket2 = "0.5.3"
 soketto = "0.7"
-solana_rbpf = "=0.6.0"
+solana_rbpf = { git = "https://github.com/solana-labs/rbpf", branch = "main" }
```

Comment:
Is this how this PR will be merged, or you're planning to change this to a specific version (0.7.0)?

Reply:
No it is not, just for CI before I publish the RBPF release. Thus, the commit is marked as work-in-progress (WIP).